pages/establishment/dashboard/views/index.jsx

```diff
@@ -63,7 +63,10 @@ const Index = ({
             }
             {
               asruAdmin && establishment.asru.filter(p => p.asruInspector).map(asru => (
-                <Link key={`${asru.id}`} page="global.profile" profileId={asru.id} label={`${asru.firstName} ${asru.lastName}`} />
+                <Fragment key={`${asru.id}`}>
+                  <Link page="global.profile" profileId={asru.id} label={`${asru.firstName} ${asru.lastName}`} />
+                  <br />
+                </Fragment>
```

Comment:
hate to be _that guy_ but seems like an unnecessary use of `<Fragments>` and `<br />`'s - can't we wrap them in <p> tags?

Reply:
if wrapping it in a `p` doesn't look right I'd prefer to just add some css to the Link elements

Reply:
ooh - my <p> was rendered as HTML